ruby/install/run.sh

```diff
@@ -7,15 +7,26 @@ if [ ! ${IS_COLOR_SOURCED} ]; then
 fi
 
 echo "${GREEN}Installing... ${BROWN}ruby...${NC}"
-gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
-curl -sSL https://get.rvm.io | bash -s stable --ruby
-gem install grb --no-document
+
+echo "${RED}Which ruby version manager would you like to use? (See readme for details) ${GREEN}[rbenv,rvm] default: rbenv${NC}"
+read choice
+
+case $choice in
+    rvm) source ${DIR}/install_rvm.sh;;
+    rbenv) source ${DIR}/install_rbenv.sh ;;
+    *) source ${DIR}/install_rbenv.sh ;;
+esac
+
+gem install bundler
+gem install grb
 gem install cocoapods
 gem install xcode-install
 gem install slather
-gem install compass
 gem install xcpretty
 gem install fastlane
-gem install appium
-gem install carthage
-gem install carthage_cache
+source ${DIR}/install_rails.sh
```

Comment:
maybe make a flag/user selection so use can select is this for mobile development or web development

rails gets installed with web
everything in the original file gets installed with mobile, except possible xcode-install

Reply:
this flag might be useful at the master script level that exposes an ENV for all other sh scripts

Reply:
I removed carthage because its no longer a gem, appium isn't either. Carthage_cache still is but not sure if people maintain it. I'm 80% sure you guys only do cocoa pods anyway at this point. :)

Or do you want to keep rails off the natives env?

Reply:
ah i see appium needs to move to npm and carthage to brew. thanks